### PR TITLE
'Gittis' Settlement Area - scrnshot edit ++ QUEST200 'the' Master standardization

### PR DESCRIPTION
**Pull request by vtange**
first pass. Spadow and others free to look over when they get here

renamed "gitise/gitese" to Gitis pertaining to IMC's wiki

EDIT: Many lines in Quest200 had bad master standardization and wrong genders. Evoniphon's name was also incorrectly spelled

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -3789,49 +3789,49 @@ QUEST_LV_0200_20150323_003788	Oh my god, so you are the one who saved me.{nl}I a
...
-QUEST_LV_0200_20150323_003831	The other soldiers went to defeat Gorgon and I can't leave this place.{nl}The only one I can trust is you. I am counting on you.
-QUEST_LV_0200_20150323_003832	Really? Thanks..{nl}The location is the joint arable land. I am counting on you!
+QUEST_LV_0200_20150323_003830	$The deadline for the farm's construction is approaching,{nl} but the monsters have suddenly became vicious, so the workers can't even touch the site.
+QUEST_LV_0200_20150323_003831	$The other soldiers went to defeat Gorgon, and I can't leave this place.{nl}I can only trust you now for this. I am counting on you.
+QUEST_LV_0200_20150323_003832	$Really? Thanks..{nl}The location is the Joint Farmland. I am counting on you!
 QUEST_LV_0200_20150323_003833	Do you think this makes sense?{nl}Those monsters used to be scared of humans, but they now attack humans with their poison.
-QUEST_LV_0200_20150323_003834	If it weren't you, Uska must have scolded me.. Thank you so much.{nl}Anyways, how come there is no order of return?
+QUEST_LV_0200_20150323_003834	$If it weren't you, Uska must have scolded me.. Thank you so much.{nl}Anyways, I wonder why there aren't any news of the soldiers' return?
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Uska -> Sir Uska?

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -4458,12 +4458,12 @@ QUEST_LV_0200_20150323_004457	Time for farewell
 QUEST_LV_0200_20150323_004458	Secure the farmland
 QUEST_LV_0200_20150323_004459	I will look for the soldiers
 QUEST_LV_0200_20150323_004460	Tell him that it will be alright
-QUEST_LV_0200_20150323_004461	The farmland of Louise(1)
+QUEST_LV_0200_20150323_004461	$Louise's Farmland(1)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Louise's Farmland (1)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -4458,12 +4458,12 @@ QUEST_LV_0200_20150323_004457	Time for farewell
...
 QUEST_LV_0200_20150323_004462	I will defeat the monsters on behalf of him
-QUEST_LV_0200_20150323_004463	About the settlement land of Gitese
-QUEST_LV_0200_20150323_004464	The farmland of Louise(2)
-QUEST_LV_0200_20150323_004465	The farmland of Louise(3)
-QUEST_LV_0200_20150323_004466	The farmland of Louise(3)
+QUEST_LV_0200_20150323_004463	$About the settlement land of Gitis
+QUEST_LV_0200_20150323_004464	$Louise's Farmland(2)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Louise's Farmland (2)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -4458,12 +4458,12 @@ QUEST_LV_0200_20150323_004457	Time for farewell
...
+QUEST_LV_0200_20150323_004465	$Louise's Farmland(3)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Louise's Farmland (3)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 5]
```diff
@@ -4458,12 +4458,12 @@ QUEST_LV_0200_20150323_004457	Time for farewell
...
+QUEST_LV_0200_20150323_004466	$Louise's Farmland(3)
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Louise's Farmland (3)

**(add a space)**

File: QUEST_LV_0200.tsv [hunk 6]
```diff
@@ -4776,9 +4776,9 @@ QUEST_LV_0200_20150323_004775	You need a sacrifice in order to receive the bless
 QUEST_LV_0200_20150323_004776	You've contributed enough monsters as sacrifices. Pray to receive the blessings.
 QUEST_LV_0200_20150323_004777	{memo X}Monsters rushed in as you prayed in front of the sanctum. You better defeat the monsters first.
 QUEST_LV_0200_20150323_004778	We should look for a way to purify the sanctum. Go meet the Priest Master who is the manager of the sanctum.
-QUEST_LV_0200_20150323_004779	We should first purify the polluted sanctum on Pamaishi Hills. Talk to the priest master what to do.
+QUEST_LV_0200_20150323_004779	We should first purify the polluted sanctum on Pamaishi Hills. Talk to the Priest Master what to do.
 QUEST_LV_0200_20150323_004780	Purify the sanctum using the divine water
-QUEST_LV_0200_20150323_004781	Priest Master asked you to purify the sanctum on Pamaishi Hills using the divine water. Go back to the Forest of Prayer and purify the polluted sanctum.
+QUEST_LV_0200_20150323_004781	The Priest Master asked you to purify the sanctum on Pamaishi Hills using the divine water. Go back to the Forest of Prayer and purify the polluted sanctum.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Pamaishi Hills -> Pamaish Hill

File: QUEST_LV_0200.tsv [hunk 7]
```diff
@@ -4776,9 +4776,9 @@ QUEST_LV_0200_20150323_004775	You need a sacrifice in order to receive the bless
 QUEST_LV_0200_20150323_004776	You've contributed enough monsters as sacrifices. Pray to receive the blessings.
 QUEST_LV_0200_20150323_004777	{memo X}Monsters rushed in as you prayed in front of the sanctum. You better defeat the monsters first.
 QUEST_LV_0200_20150323_004778	We should look for a way to purify the sanctum. Go meet the Priest Master who is the manager of the sanctum.
-QUEST_LV_0200_20150323_004779	We should first purify the polluted sanctum on Pamaishi Hills. Talk to the priest master what to do.
+QUEST_LV_0200_20150323_004779	We should first purify the polluted sanctum on Pamaishi Hills. Talk to the Priest Master what to do.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Pamaishi Hills -> Pamaish Hill

File: QUEST_LV_0200.tsv [hunk 8]
```diff
@@ -4799,12 +4799,12 @@ QUEST_LV_0200_20150323_004798	You've completed repairing the statue. Look closel
...
 QUEST_LV_0200_20150323_004800	The statue which you spent the time repairing it was fake. Break it fast since you don't know what will happen at the demon summon field.
 QUEST_LV_0200_20150323_004801	The sanctum is already polluted. Defeat the monsters that are coming to the polluted sanctum.
-QUEST_LV_0200_20150323_004802	The manager of this sanctum is Cleric Master. Ask him what he should do to purify the pollution of the sanctum.
-QUEST_LV_0200_20150323_004803	Listen to the advices of Cleric Master
+QUEST_LV_0200_20150323_004802	The manager of this sanctum is the Cleric Master. Ask him what he should do to purify the pollution of the sanctum.
+QUEST_LV_0200_20150323_004803	Listen to the advices of the Cleric Master
 QUEST_LV_0200_20150323_004804	Get some advices from the Cleric Master regarding the polluted sanctum at the Forest of Prayer.
-QUEST_LV_0200_20150323_004805	You've obtained an advise on the way to purify the polluted sanctum from the cleric master. Go back to the Forest of Prayer.
+QUEST_LV_0200_20150323_004805	You've obtained an advise on the way to purify the polluted sanctum from the Cleric Master. Go back to the Forest of Prayer.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: You've obtained an advise -> You've obtained advise?
Go back to the Forest of Prayer. -> Return to the Forest of Prayer

File: QUEST_LV_0200.tsv [hunk 9]
```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
...
 QUEST_LV_0200_20150323_005236	The disciple, Prosit can't forgive the demons that are ruining the great cathedral. Defeat the demons on behalf of the disciple, Prosit.
 QUEST_LV_0200_20150323_005237	You've defeated many demons. Talk to the disciple, Prosit.
-QUEST_LV_0200_20150323_005238	Talk to Gitise
-QUEST_LV_0200_20150323_005239	You can see Gitise who is worried a lot at the settlement land of Gitese. Talk with Gitese.
-QUEST_LV_0200_20150323_005240	Ask for help from Pyromancer Master at Klaipeda
-QUEST_LV_0200_20150323_005241	Gitese told you that the workers at the settlement land were attacked by the monsters and they are sick. Talk to Pyromance Master to get his help.
-QUEST_LV_0200_20150323_005242	Pyromancer Master told you that he will definitely help you. Talk to Pyromancer Master again.
+QUEST_LV_0200_20150323_005238	$Talk to Gitis
+QUEST_LV_0200_20150323_005239	$You can see Gitis who is worried a lot at the settlement land of Gitis. Talk with Gitis.
+QUEST_LV_0200_20150323_005240	$Ask for help from the Pyromancer Master at Klaipeda
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: at Klaipeda -> in Klaipeda

File: QUEST_LV_0200.tsv [hunk 10]
```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
...
+QUEST_LV_0200_20150323_005241	$Gitis told you that the workers at the settlement land were attacked by the monsters and they are sick. Talk to the Pyromancer Master to get her help.
+QUEST_LV_0200_20150323_005242	$The Pyromancer Master told you that he will definitely help you. Talk to the Pyromancer Master again.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: he -> she

File: QUEST_LV_0200.tsv [hunk 11]
```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
...
 QUEST_LV_0200_20150323_005243	Collect the samples of pink Chuparukas at the settlement land
 QUEST_LV_0200_20150323_005244	{memo X}Pyromancer Master told you that he will definitely help you. First get some samples of pink Chuparukas at the settlement land.
-QUEST_LV_0200_20150323_005245	Hand it over to Pyromancer Master
-QUEST_LV_0200_20150323_005246	You've obtained the samples of pink Chuparukas. Hand them over to Pyromancer Master.
+QUEST_LV_0200_20150323_005245	$Hand it over to the Pyromancer Master
+QUEST_LV_0200_20150323_005246	$You've obtained the samples of pink Chuparukas. Hand them over to the Pyromancer Master.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: pink Chuparukas -> Pink Chupalukas

File: QUEST_LV_0200.tsv [hunk 12]
```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
...
 QUEST_LV_0200_20150323_005247	Obtain %s from the settlement land
-QUEST_LV_0200_20150323_005248	Pyromancer Master who was investigating the samples of the monsters doesn't feel good about something. Talk to Pyromancer Master.
+QUEST_LV_0200_20150323_005248	$The Pyromancer Master who was investigating the samples of the monsters doesn't feel good about something. Talk to the Pyromancer Master.
 QUEST_LV_0200_20150323_005249	Search for the clues of the massive chaos at the settlement land
-QUEST_LV_0200_20150323_005250	Pyromancer Master can feel some unknown poison and the spell. Go back to the settlement land and find out the true nature of the poison and the spell.
-QUEST_LV_0200_20150323_005251	You've found a leather and a dagger with the magic field drawn in them at Nuodo Bank. Take them to Pyromancer Master.
+QUEST_LV_0200_20150323_005250	$The Pyromancer Master can feel some unknown poison and the spell. Go back to the settlement land and find out the true nature of the poison and the spell.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: - The Pyromancer Master can feel some unknown poison and the spell -> The Pyromancer Master can feel some unknown poison and magic?
- true nature of the poison and the spell. -> true nature of the poison and the magic?
- Go back to the settlement ... -> Return to the settlement...

File: QUEST_LV_0200.tsv [hunk 13]
```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
...
+QUEST_LV_0200_20150323_005251	$You've found a leather and a dagger with the magic field drawn in them at Nuodo Bank. Take them to the Pyromancer Master.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: magic field -> magic circle
Nuodo Bank -> Nuodo Coast

File: QUEST_LV_0200.tsv [hunk 14]
```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
...
 QUEST_LV_0200_20150323_005252	Destroy the unknown altar
-QUEST_LV_0200_20150323_005253	Unfortunately, Pyromancer Master doesn't know well about the spell. Talk to Pyromancer Master again.
-QUEST_LV_0200_20150323_005254	Talk to Bokor Master about the true nature of the spell.
-QUEST_LV_0200_20150323_005255	Bokor Master knows well about the spells. Ask Bokor Master about the true nature of the spells.
-QUEST_LV_0200_20150323_005256	Bokor Master told you that the spell spreads the poison that is on the the dagger. But, he told you that he doesn't know well about the dagger so told you to meet Ranger Master.
-QUEST_LV_0200_20150323_005257	Talk to Gitise
+QUEST_LV_0200_20150323_005253	$Unfortunately, the Pyromancer Master doesn't know well about the spell. Talk to the Pyromancer Master again.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:5253]: the spell -> the magic?

File: QUEST_LV_0200.tsv [hunk 15]
```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
...
+QUEST_LV_0200_20150323_005254	$Talk to the Bokor Master about the true nature of the spell.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:5254]: of the spell -> of the magic?

File: QUEST_LV_0200.tsv [hunk 16]
```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
...
+QUEST_LV_0200_20150323_005255	$The Bokor Master knows well about the spells. Ask the Bokor Master about the true nature of the spell.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:5255]: about the spells -> about magic?
true nature of the spell -> true nature of the magic?

File: QUEST_LV_0200.tsv [hunk 17]
```diff
@@ -5235,30 +5235,30 @@ QUEST_LV_0200_20150323_005234	The disciple, Goda seems to have lots of complaint
...
+QUEST_LV_0200_20150323_005256	$The Bokor Master told you that the spell spreads the poison that is on the the dagger. But, she told you that she doesn't know well about the dagger so told you to meet the Ranger Master.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:5256]: the spell -> the magic

File: QUEST_LV_0200.tsv [hunk 18]
```diff
@@ -5275,25 +5275,25 @@ QUEST_LV_0200_20150323_005274	Look for the soldiers who went out to defeat Gorgo
...
-QUEST_LV_0200_20150323_005278	Louise needs your help. Talk to Louise near the Louise Farm.
-QUEST_LV_0200_20150323_005279	Defeat Ridimed at Louise Farm
-QUEST_LV_0200_20150323_005280	Louise told you that he can't work at all due to Ridimed that came into his farmland. Defeat Ridimed at Louise farmland.
-QUEST_LV_0200_20150323_005281	You've defeated many Ridimeds. Report to Louise.
-QUEST_LV_0200_20150323_005282	Defeat Ridimed at Louise Farm
+QUEST_LV_0200_20150323_005278	$Louise needs your help. Talk to Louise near the Louise's Farm.
+QUEST_LV_0200_20150323_005279	$Defeat Ridimed at Louise's Farm
+QUEST_LV_0200_20150323_005280	$Louise told you that he can't work at all due to the Ridimed that came into his farmland. Defeat the Ridimed at Louise's farmland.
+QUEST_LV_0200_20150323_005281	$You've defeated many Ridimeds. Report to Louise.
+QUEST_LV_0200_20150323_005282	$Defeat Ridimed at Louise's Farm
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Farm -> Farmland

File: QUEST_LV_0200.tsv [hunk 19]
```diff
@@ -5275,25 +5275,25 @@ QUEST_LV_0200_20150323_005274	Look for the soldiers who went out to defeat Gorgo
...
 QUEST_LV_0200_20150323_005276	Defeat Gorgon
 QUEST_LV_0200_20150323_005277	Talk to Louise
-QUEST_LV_0200_20150323_005278	Louise needs your help. Talk to Louise near the Louise Farm.
-QUEST_LV_0200_20150323_005279	Defeat Ridimed at Louise Farm
-QUEST_LV_0200_20150323_005280	Louise told you that he can't work at all due to Ridimed that came into his farmland. Defeat Ridimed at Louise farmland.
-QUEST_LV_0200_20150323_005281	You've defeated many Ridimeds. Report to Louise.
-QUEST_LV_0200_20150323_005282	Defeat Ridimed at Louise Farm
+QUEST_LV_0200_20150323_005278	$Louise needs your help. Talk to Louise near the Louise's Farm.
+QUEST_LV_0200_20150323_005279	$Defeat Ridimed at Louise's Farm
+QUEST_LV_0200_20150323_005280	$Louise told you that he can't work at all due to the Ridimed that came into his farmland. Defeat the Ridimed at Louise's farmland.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Louise's farmland -> Louise's Farmland?

File: QUEST_LV_0200.tsv [hunk 20]
```diff
@@ -5275,25 +5275,25 @@ QUEST_LV_0200_20150323_005274	Look for the soldiers who went out to defeat Gorgo
...
+QUEST_LV_0200_20150323_005288	$Louise made the fences within a short period of time with the wood. Talk to Louise again.
+QUEST_LV_0200_20150323_005289	$Set up the fences on Louise's Farmland
+QUEST_LV_0200_20150323_005290	$Louise told you that there are still many monsters so asked you to set up the completed fences. Set up the fences on Louise's Farmland.
+QUEST_LV_0200_20150323_005291	$You've set up all the fences. Talk to Louise.
 QUEST_LV_0200_20150323_005292	Obtain %s that will be used as the materials for the scracrow
 QUEST_LV_0200_20150323_005293	Obtain the materials that are lacking by defeating the monsters
 QUEST_LV_0200_20150323_005294	It seems that something is not going well for Gares. Talk to Gares.
 QUEST_LV_0200_20150323_005295	Collect the samples of Shakmoli
-QUEST_LV_0200_20150323_005296	Gares told you that the rate at the crops are growing is not fast so he wants to know why. Please obtain the samples of orange Shakmolis.
+QUEST_LV_0200_20150323_005296	$Gares told you that the rate at which the crops are growing is not as fast as usual, so he wants to know why. Please obtain the samples of orange Shakmolis.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: orange -> Orange

File: QUEST_LV_0200.tsv [hunk 21]
```diff
@@ -6749,7 +6749,7 @@ QUEST_LV_0200_20150414_006752	We really owe you one.
 QUEST_LV_0200_20150414_006753	Thank you.{nl}I will find all ways to avoid opening the contaminated gap.
 QUEST_LV_0200_20150414_006754	How long can Aldona keep..
 QUEST_LV_0200_20150414_006755	If the Vakarine has received the Laima's warning in advance..{nl}The situation is better than now?
-QUEST_LV_0200_20150414_006756	Even I received the land from Gitise but it's all dirty.{nl}Especially, Ridimed. But, what can he do with the hand just got the pick?
+QUEST_LV_0200_20150414_006756	$I received land from Gitis, but it's all dirty.{nl}The Ridimeds are the problem. But, what can I do with just a hoe and pick?
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:6752]: The Ridimeds are the problem -> The Ridimed is the problem

_(There's no multiple Ridimeds, I believe its referring to only one Ridimed)_

File: QUEST_LV_0200.tsv [hunk 22]
```diff
@@ -6841,7 +6841,7 @@ QUEST_LV_0200_20150414_006844	Find out someone who knows the several fragments.
 QUEST_LV_0200_20150414_006845	Talk to Jonas
 QUEST_LV_0200_20150414_006846	Regain the Padeka Altar
 QUEST_LV_0200_20150414_006847	All Place of Sanctuary
-QUEST_LV_0200_20150414_006848	Pyromancer Master said he'd gladly help us. Firstly, collect the sample of Pink Chupaluka in the Pioneer Place.
+QUEST_LV_0200_20150414_006848	The Pyromancer Master said she'd gladly help us. Firstly, collect the sample of Pink Chupaluka in the Pioneer Place.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:6844]: Pioneer Place? Isn't it Settlement Area?

File: QUEST_LV_0200.tsv [hunk 23]
```diff
@@ -8561,10 +8561,10 @@ QUEST_LV_0200_20150714_008564	Talk with Hunter, Reina
 QUEST_LV_0200_20150714_008565	Mintz said Hunter Reina would be able to find an adequate evidence. Go look for Hunter Reina at Rikdimo Gate Route.
 QUEST_LV_0200_20150714_008566	Hunter Reina looks ill. Talk to her first.
 QUEST_LV_0200_20150714_008567	Search for the traps using Jareth
-QUEST_LV_0200_20150714_008568	Hunter Reina lent you her companion, Jareth since she can't move. Use Jareth to look for Ebonippon's trap.
-QUEST_LV_0200_20150714_008569	You found the Ebonippon's trap. Go back to Reina with Jareth.
+QUEST_LV_0200_20150714_008568	Hunter Reina lent you her companion, Jareth since she can't move. Use Jareth to look for Evoniphon's trap.
+QUEST_LV_0200_20150714_008569	You found the Evoniphon's trap. Go back to Reina with Jareth.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: the Evoniphon -> Evoniphon
Go Back -> Return

File: QUEST_LV_0200.tsv [hunk 24]
```diff
@@ -7858,8 +7858,8 @@ QUEST_LV_0200_20150714_007861	Okay. I will discuss with the father superior.{nl}
 QUEST_LV_0200_20150714_007862	I will wait for their decision here first.{nl}If their declaration doesn't get withdrawn, then I should think about what to do at then.
 QUEST_LV_0200_20150714_007863	So it seems that their discussion is over.{nl}Let's go there and listen to their conclusion.
 QUEST_LV_0200_20150714_007864	We haven't finished our discussion.{nl}Please wait with the hunters.
-QUEST_LV_0200_20150714_007865	After we've discussed with the senior monks...{nl}We've decided to withdraw our declaration to protect Ebonippon.
-QUEST_LV_0200_20150714_007866	From this point on, Ebonippon won't be protected by Tila Monastery and we request Ebonippon to leave at once.{nl}Also, we are not going to be involved with others' decision to punish him.
+QUEST_LV_0200_20150714_007865	After we've discussed with the senior monks...{nl}We've decided to withdraw our declaration to protect Evoniphon.
+QUEST_LV_0200_20150714_007866	From this point on, Evoniphon won't be protected by Tila Monastery and we request Evoniphon to leave at once.{nl}Also, we are not going to be involved with others' decision to punish him.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: Tila Monastery -> Tyla Monastery
(_the Tyla Monastery_ sounds better)

File: QUEST_LV_0200.tsv [hunk 25]
```diff
@@ -8703,13 +8703,13 @@ QUEST_LV_0200_20150714_008706	Call out the father superior.
...
-QUEST_LV_0200_20150714_008710	The father superior moved out from the room. Ask him to withdraw the decision to protect Ebonippon.
+QUEST_LV_0200_20150714_008710	The father superior moved out from the room. Ask him to withdraw the decision to protect Evoniphon.
 QUEST_LV_0200_20150714_008711	The father superior won't decide quickly. While he discusses with senior monks, go meet the commander, Mintz.
-QUEST_LV_0200_20150714_008712	After listening from the senior monks, the father superior finally discovered the trun nature of Ebonippon. Talk to him.
-QUEST_LV_0200_20150714_008713	It seems that the decision about what to do with Ebonippon has been confirmed. Talk to the father superior.
-QUEST_LV_0200_20150714_008714	Search for the assassin, Ebonippon.
-QUEST_LV_0200_20150714_008715	The father superior requested you to kick out Ebonippon from the monastery. Look for Ebonippon first in the monastery.
-QUEST_LV_0200_20150714_008716	Unfortunately, Ebonippon ran away and Mintz is injured. Talk with Mintz.
+QUEST_LV_0200_20150714_008712	After listening from the senior monks, the father superior finally discovered the trun nature of Evoniphon. Talk to him.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv]: trun nature -> true nature

File: QUEST_LV_0200.tsv (hunk 14 above)
Comment (vtange) [inline QUEST_LV_0200.tsv:5253]: it's a dagger with a poison-spreading spell cast on it. for consistency's sake 'spell' is probably better.

File: QUEST_LV_0200.tsv (hunk 21 above)
Comment (vtange) [inline QUEST_LV_0200.tsv:6752]: There are actually lots of Ridimeds :) He literally tells you this line while like 5 of them waddle around you and him

Comment (Sunaries) [inline QUEST_LV_0200.tsv:6752]: Ah, I see. I haven't progressed that far. Thanks.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**